spare5/resources/jobs.py

```diff
@@ -1,18 +1,34 @@
+from .base import Resource
 from .base import ListResource
+from .base import MethodNotAllowedException
 
+from .responses import Responses
 
-class Job(object):
+
+class Job(Resource):
     STAR_RATING = 'STARRATING'
 
     JOB_TYPES = (STAR_RATING,)
 
+    def update(self, **kwargs):
+        raise MethodNotAllowedException()
+
+    @property
+    def responses(self):
+        return Responses(self.client, self)
+
 
 class Jobs(ListResource):
     REQUIRED_PARAMS = ('num_responders',)
+    _resource_class = Job
+
+    def __init__(self, client, batch, result=None):
```

Comment:
It's weird that the sig here doesn't match that of the parent class - I don't understand why if it was intentional.